client/scss/_tools.breakpoints.scss

```diff
@@ -1,5 +1,5 @@
-$breakpoint-small: $breakpoint-mobile - 0.0625em;
-$breakpoint-medium: $breakpoint-mobile;
+$breakpoint-small: breakpoint-max(xs);
+$breakpoint-medium: breakpoint-min(sm);
 
 @mixin small {
```

Comment:
An after-thought maybe we should replace these mixins with the new introduced mixins. What are you thoughts on this?

Reply:
Yes!

Reply:
In addition I also renamed and check the utility and visibility classes to have a consistent naming with the breakpoints. So `@small` became `@xs` and `@medium` became `@sm`.